docs/AUTHENTICATION.md

```diff
@@ -0,0 +1,11 @@
+Authentication
+===
+
+1. frontend: Sends user to the lock interface that's provided by auth0 when required.
+1. frontend: Lock interface returns a json web token.
```

Comment:
Could you link to an example of this json? It's nice to know what properties the return object is supposed to have